src/transformers/integrations/integration_utils.py

```diff
@@ -816,18 +887,30 @@ def on_save(self, args, state, control, **kwargs):
                 for k, v in dict(self._wandb.summary).items()
                 if isinstance(v, numbers.Number) and not k.startswith("_")
             }
+            checkpoint_metadata["model/num_parameters"] = self._wandb.config["model/num_parameters"]
 
             ckpt_dir = f"checkpoint-{state.global_step}"
             artifact_path = os.path.join(args.output_dir, ckpt_dir)
             logger.info(f"Logging checkpoint artifacts in {ckpt_dir}. ...")
             checkpoint_name = (
-                f"checkpoint-{self._wandb.run.id}"
+                f"model-{self._wandb.run.id}"
                 if (args.run_name is None or args.run_name == args.output_dir)
-                else f"checkpoint-{self._wandb.run.name}"
+                else f"model-{self._wandb.run.name}"
```

Comment:
is that not breaking? could we keep model and `aliases=["model"]` ?

Reply:
> is that not breaking? could we keep model and `aliases=["model"]` ?

@ArthurZucker : This doesn't break existing functionality. Currently, we create separate Weights & Biases Artifacts for models and checkpoints. However, this results in multiple artifacts for the same experiment. This instead creates a single artifact with model and checkpoint aliases, allowing the users to log models and checkpoints to the same artifact.